multisig/proposals/ethereum/vip-019/index.ts

```diff
@@ -0,0 +1,112 @@
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { ethereum } = NETWORK_ADDRESSES;
+
+export const WeETH_ORACLE_EQUIVALENCE = "0xEa687c54321Db5b20CA544f38f08E429a4bfCBc8";
+export const weETH = "0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee";
+export const CHAINLINK_WEETH_FEED = "0x5c9C449BbC9a6075A2c061dF312a35fd1E05fF22";
+export const VTREASURY = "0xfd9b071168bc27dbe16406ec3aba050ce8eb22fa";
+export const vweETH = "0xb4933AF59868986316Ed37fa865C829Eba2df0C7";
+export const REWARDS_DISTRIBUTOR = "0xDCB0CfA130496c749738Acbe2d6aA06C7C320f06";
+export const COMPTROLLER = "0xF522cd0360EF8c2FF48B648d53EA1717Ec0F3Ac3";
+export const USDC = "0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48";
+
+// 24 hours stale period as per https://data.chain.link/feeds/ethereum/mainnet/weeth-eth
+export const CHAINLINK_STALE_PERIOD = "86400";
```

Comment:
To have some margin, we usually set a higher stale period than the heartbeat. For feeds with heartbeats of 24h, we have set stale periods of 26h in the past ([example](https://github.com/VenusProtocol/vips/blob/develop/multisig/proposals/ethereum/vip-001/index.ts#L11)).

I would set a stale period of 26 hours.